frontend/components/notifications/NotificationsClientsTable.vue

```diff
@@ -0,0 +1,160 @@
+<script setup lang="ts">
+import { useNotificationsClientsStore } from '~/stores/notifications/useNotificationsClientsStore';
+import type { Cursor } from '~/types/datatable'
+
+defineEmits<{ (e: 'openDialog'): void }>()
+
+const { width } = useWindowSize()
+const cursor = ref<Cursor>()
+const pageSize = ref<number>(10)
+const { t: $t } = useTranslation()
+
+const {
+  clientsNotifications,
+  isLoading,
+  onSort,
+  query,
+  setCursor,
+  setPageSize,
+  setSearch,
+} = useNotificationsClientsStore()
+
+const colsVisible = computed(() => {
+  return {
+    footer: 1024,
+    threshold: width.value > 830,
+  }
+})
+
+const clientLink = computed(() => {
+  // TODO: implement client link when it's available from the API
+  // TODO: Test the endpoints
+  // return `/client/${data.clientId}`
+  return '/notifications'
+})
+</script>
+
+<template>
+  <div>
+    <BcTableControl
+      :title="$t('notifications.clients.title')"
+      :search-placeholder="$t('notifications.dashboards.search_placeholder')"
+      @set-search="setSearch"
+    >
+      <template #table>
+        <ClientOnly fallback-tag="span">
+          <BcTable
+            :data="clientsNotifications"
+            data-key="notification_id"
+            :cursor
+            :page-size
+            :selected-sort="query?.sort"
+            :loading="isLoading"
+            :add-spacer="true"
+            @set-cursor="setCursor"
+            @sort="onSort"
+            @set-page-size="setPageSize"
+          >
+            <Column
+              field="machine_name"
+              sortable
+              header-class="col-client-name"
+              body-class="col-client-name"
+              :header="$t('notifications.clients.col.client_name')"
+            >
+              <template #body="slotProps">
+                <pre>
+                  {{ slotProps.data.client_name }}
+                </pre>
+              </template>
+            </Column>
+            <Column
+              field="machine_name"
+              header-class="col-client-name"
+              body-class="col-client-name"
+              :header="$t('notifications.clients.col.version')"
+            >
+              <template #body="slotProps">
+                <BcLink
+                  :to="clientLink"
+                  class="link link-dashboard"
+                  target="_blank"
+                >
+                  {{ slotProps.data.version }}
+                </BcLink>
```

Comment:
plz remove link functionality if there is no target yet... @LuccaBitfly can you tell us where the `data` is supposed to come from?

Reply:
-> BEDS-475